File: common/src/main/java/de/invees/portal/common/model/v1/product/price/ProductPriceV1.java
package de.invees.portal.common.model.v1.product.price;

import de.invees.portal.common.model.Model;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
public class ProductPriceV1 implements Model {

  private double amount;
  private List<OneOffProductPriceV1> oneOffList;

}

File: model/v1/product/price/ProductPriceV1.ts
import IntervalProductPriceV1 from "@/model/v1/product/price/IntervalProductPriceV1.ts";
import OneOffProductPriceV1 from "@/model/v1/product/price/OneOffProductPriceV1.ts";

export default interface ProductPriceV1 {
  intervalList: Array<IntervalProductPriceV1>;
  oneOffList: Array<OneOffProductPriceV1>;
}
